coreos-kubernetes-multimaster/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "sshKeyData": {
+      "value": "GEN-SSH-PUB-KEY"
+    },
+    "tenantId":{
+      "value": "GEN-UNIQUE"
```

Comment:
I don't think GEN-UNIQUE will necessarily generate values that could be valid tenant or client IDs.